server/cmd/gram/start.go

```diff
@@ -423,6 +425,10 @@ func newStartCommand() *cli.Command {
 				sessionManager = s
 			}
 
+			// Initialize chat sessions manager
+			// TODO: Should this use a different encryption key?
+			chatSessionsManager := chatsessions.NewManager(logger, redisClient, c.String("encryption-key"))
```

Comment:
IMO it shouldn't share encryption-key for this. There should be a JWT_SIGNING key